packages/app-staking/src/index.tsx

```diff
@@ -25,18 +24,17 @@ import translate from './translate';
 
 type Props = AppProps & ApiProps & I18nProps & {
   allAccounts?: SubjectInfo,
-  balances?: DerivedBalancesMap,
-  session_validators?: Array<AccountId>,
-  staking_controllers?: [Array<AccountId>, Array<Option<AccountId>>],
+  allStashesAndControllers?: [Array<AccountId>, Array<Option<AccountId>>],
```

Comment:
See the `Array<T>` comment (not a nit, just something we should probably do, but if not, no trainsmash either, it will get done)